plugins/SitesManager/SitesManager.php

```diff
@@ -39,6 +39,36 @@ class SitesManager extends \Piwik\Plugin
     public const KEEP_URL_FRAGMENT_YES = 1;
     public const KEEP_URL_FRAGMENT_NO = 2;
 
+    public const COMMON_URL_PARAMS_TO_EXCLUDE =  ['creditcardnumber', 'off', 'kreditkarte', 'debitcard', 'kreditkort',
```

Comment:
For flexibility purpose we could move that list to global config. This would also allow to change the list

Reply:
I've added this to the global config now, good suggestion